Sources/Error Handling/ErrorUtils.swift

```diff
@@ -544,6 +544,19 @@ enum ErrorUtils {
                                 fileName: fileName, functionName: functionName, line: line)
     }
 
+    /**
+     * Constructs an Error with the ``ErrorCode/invalidSubscriptionOffer`` code.
+     */
+    static func invalidSubscriptionOfferError(
```

Comment:
I'm probably missing something but I can't find where this error is used 🤔

Reply:
Ahh. This was used in the `purchase(sk2Product:winbackOffer:)` function that we removed earlier in the review process. Good catch! 😄

I've removed this in 08ebe4ff8ac6fb5862c123489c0085fc7ea6621f. We might want to bring back something similar when implementing win-backs into the purchase params, but we can cross that bridge when we get there 😄